clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/deploy/description/CreateServerGroupDescription.java

```diff
@@ -31,8 +32,8 @@
 public class CreateServerGroupDescription extends AbstractECSDescription {
   String ecsClusterName;
   String iamRole;
-  Integer containerPort;
-  String targetGroup;
+  @Deprecated Integer containerPort;
```

Comment:
Sounds good! I added the deprecated annotation to prevent anyone from making changes in the future to use that field. If there's another contributor that decides they want to add some additional logic to use those properties, it'll allow them to see that the field is scoped for removal (even if it's an year before we remove it)

Reply:
Looks like this is https://github.com/spinnaker/clouddriver/blob/master/clouddriver-cloudfoundry/src/main/java/com/netflix/spinnaker/clouddriver/cloudfoundry/security/CloudFoundryCredentials.java#L51

I can add a comment, but the Javadoc comments are usually on methods.